apps/host/src/components/modules/challenges/challenge-grid/challenge-filter.tsx

```diff
@@ -67,7 +73,28 @@ const ChallengeFilters = ({
         optionSelected={selectedDifficulties}
         setOptionSelected={(val: OptionType[]) => setSelectedDifficulties(val)}
       />
-
+      <div className={styles.customSegment}>
+        <button
+          data-active={isSegmentBtn1 ? true : false}
```

Comment:
`data-active={isSegmentBtn1}` does it work?

Reply:
yes its working, I am using Css based on this 
<img width="784" alt="Screenshot 2024-07-14 at 5 30 57 PM" src="https://github.com/user-attachments/assets/8fc3ae5b-940b-45e6-939b-5faad345d678">